src/EnergyPlus/MixedAir.cc

```diff
@@ -4146,6 +4267,10 @@ namespace MixedAir {
 			OAController( OAControllerNum ).OAMassFlow = min( OAController( OAControllerNum ).OAMassFlow, OAController( OAControllerNum ).MaxOAMassFlowRate );
 		}
 
+		// Update demand managers here and before EMS override becuase we want EMS to have higher priority than demand manager
+		if ((OAController(OAControllerNum).ManageDemand) && (OAController(OAControllerNum).OAMassFlow > OAController(OAControllerNum).DemandLimitFlowRate))
```

Comment:
I need further clarification on this. Are you saying that I should include one more if before this to examine if current loop is in Simulation, Warmup or Sizing mode?